### PR TITLE
NGSTACK-940 replace a tag with button tag on language switcher and mo…

### PR DESCRIPTION
**Pull request by TTrakostanec**
…bile navigation, add title attr on social links

### CODE REVIEW

File: translations/messages.en.yaml [hunk 1]
```diff
@@ -40,3 +40,7 @@ ngsite.job_application.responsibilities: 'Responsibilities'
 ngsite.job_application.deadline: 'Application deadline'
 
 ngsite.skip_to_main_content: 'Skip to main content'
+
+ngsite.share_on_twitter: 'Share on Twitter'
+ngsite.share_on_linkedin: 'Share on LinkedIn'
+ngsite.share_on_facebook: 'Share on Facebook'
```
Comment (emodric) [inline translations/messages.en.yaml:46]: Missing new line at EOF.

File: templates/themes/app/content/parts/social_share.html.twig [hunk 2]
```diff
@@ -2,22 +2,23 @@
...
             <li>
-                <a class="static-social-button" target="_blank" rel="nofollow" href="https://www.facebook.com/sharer.php?u={{ app.request.uri }}&amp;t={{ content.fields.title.value }}">
+                <a class="static-social-button" title="{{ 'ngsite.share_on_facebook'|trans }}" target="_blank" rel="nofollow" href="https://www.facebook.com/sharer.php?u={{ app.request.uri }}&amp;t={{ content.fields.title.value }}">
                     <i class="icon-facebook"></i>
                 </a>
             </li>
         </ul>
     </div>
 </div>
+
```
[the remark this answers is not in the document]
Comment (TTrakostanec) [reply templates/themes/app/content/parts/social_share.html.twig]: fixed in https://github.com/netgen/media-site/pull/214/commits/31049da141cfa1c807517ef467ae23ea02e929fe

File: translations/messages.en.yaml (hunk 1 above)
Comment (TTrakostanec) [reply translations/messages.en.yaml]: fixed in https://github.com/netgen/media-site/pull/214/commits/31049da141cfa1c807517ef467ae23ea02e929fe

File: templates/themes/app/pagelayout/header.html.twig [hunk 3]
```diff
@@ -2,9 +2,9 @@
     <div class="container">
         {% include '@ibexadesign/content/parts/site_logo.html.twig' %}
 
-        <a class="mainnav-toggle" href="#">
+        <button class="mainnav-toggle">
             <span class="hamburger"></span>
```
Comment (zulicek) [inline templates/themes/app/pagelayout/header.html.twig:6]: tu bi bilo dobro dodati aria-hidden="true" jer je samo dekorativni element

File: templates/themes/app/pagelayout/header.html.twig [hunk 4]
```diff
@@ -2,9 +2,9 @@
     <div class="container">
         {% include '@ibexadesign/content/parts/site_logo.html.twig' %}
 
-        <a class="mainnav-toggle" href="#">
+        <button class="mainnav-toggle">
```
Comment (zulicek) [inline templates/themes/app/pagelayout/header.html.twig]: ovdje bi bilo dobro dodati 

- aria-expanded="false" (promijeniti na klik na true)
- aria-controls="main-navigation" -> samo što se to referencira na id, a ne klasu pa treba i na taj element dodati id
- aria-label="Open main navigation"

File: templates/themes/app/pagelayout/header/language_selector.html.twig [hunk 5]
```diff
@@ -2,9 +2,9 @@
     {% set prioritized_languages = ibexa.configResolver.getParameter('languages') %}
 
     <div class="language-selector dropdown">
-        <a href="#lang-selector" data-bs-toggle="dropdown" class="current-lang">
+        <button data-bs-toggle="dropdown" class="current-lang">
```
Comment (zulicek) [inline templates/themes/app/pagelayout/header/language_selector.html.twig]: @TTrakostanec  isto kao i za mainnav-toggle:

aria-expanded="false" (promijeniti na klik na true)
aria-controls="lang-selector" -> samo što se to referencira na id, a ne klasu pa treba i na taj element dodati id
aria-label="Change language"

**Review by emodric — APPROVED** [review]

File: templates/themes/app/pagelayout/header.html.twig (hunk 4 above)
Comment (TTrakostanec) [reply templates/themes/app/pagelayout/header.html.twig]: fixed in https://github.com/netgen/media-site/pull/214/commits/f1fbfc5dc05bb50aef14f755dd47c099f632e6b7

File: templates/themes/app/pagelayout/header/language_selector.html.twig (hunk 5 above)
Comment (TTrakostanec) [reply templates/themes/app/pagelayout/header/language_selector.html.twig]: fixed in https://github.com/netgen/media-site/pull/214/commits/f1fbfc5dc05bb50aef14f755dd47c099f632e6b7

File: assets/js/components/PageHeader.component.js [hunk 6]
```diff
@@ -38,6 +38,16 @@ export default class PageHeader {
     this.navToggle.addEventListener('click', (event) => {
       event.preventDefault();
 
+      let ariaExpanded = this.navToggle.getAttribute('aria-expanded');
```
Comment (zulicek) [inline assets/js/components/PageHeader.component.js:41]: to možeš skratiti na: 
   let ariaExpanded = this.navToggle.getAttribute('aria-expanded') === 'true';

   this.navToggle.setAttribute('aria-expanded', !ariaExpanded);

**Review by emodric — CHANGES_REQUESTED** [review]
Needs rebase.

**Review by emodric — APPROVED** [review]

**Pull request merged**